app/views/projects/index.html.haml

```diff
@@ -25,8 +26,9 @@
               %br
               %nobr
                 %small
-                  forked from
+                  Forked from:
                   = link_to project.source_full_name, project.source_github_url, target: '_blank'
+          %td= project.language
```

Comment:
It this the human language the repo is mainly written in or the programming language?